Enable the detection of container-formats

FlorianO opened the pull request:
Due to the fact that nowaday many file formats are actually container formats (e.g. Office, mkv, ...) 
i wrapped the input-stream in a TikaInputStream.

The detection of information for container formats is only possible if you use TikaInputStream.
See http://tika.apache.org/1.2/detection.html#Content_Detection

This will consume more CPU and memory than the before.
So maybe take this pull-request with a grain of salt and find a good solution to furthermore 
make it possible to use the old behaviour.

FlorianO committed:
Wrap in TikaInputStream to detect ContainerFormats

alexott commented:
Hmmm, maybe it would be better implement this as function with additional flag?

FlorianO commented:
Ok, will do this the next days.

Best regards Florian

2012/11/30 Alex Ott notifications@github.com

> Hmmm, maybe it would be better implement this as function with additional
> flag?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/alexott/clj-tika/pull/2#issuecomment-10896076.

Commit:
Close TikaInputStream on detect-mime-type for InputStream

Not closing this stream can leak temporary files under certain
circumstances, e.g. when the InputStream is a zip archive.

FlorianO committed:
Merge pull request #1 from DerGuteMoritz/fix-tempfile-leak

Close TikaInputStream on detect-mime-type for InputStream

FlorianO committed:
Upgraded tika version

FlorianO committed:
changed to str

FlorianO committed:
Updated tika dependency to apache tika 1.4

FlorianO committed:
Updated tika dependency
